l10n_it_pos_fatturapa/static/src/js/ClientDetailsEdit.js

```diff
@@ -0,0 +1,29 @@
+odoo.define("l10n_it_pos_fatturapa.ClientDetailsEdit", function (require) {
+    "use strict";
+
+    const ClientDetailsEdit = require("point_of_sale.ClientDetailsEdit");
+    const Registries = require("point_of_sale.Registries");
+
+    const PosClientDetailsEditCode = (ClientDetailsEdit) =>
+        class extends ClientDetailsEdit {
+            constructor() {
+                super(...arguments);
+            }
+            captureChange(event) {
+                super.captureChange(event);
+                if (event.target.name === "electronic_invoice_subjected") {
+                    const checked = event.currentTarget.checked;
+                    this.changes[event.target.name] = Boolean(checked);
+                    $(".electronic_invoice_subjected")
+                        .toArray()
+                        .forEach(function (el) {
+                            $(el).css("display", checked ? "block" : "none");
```

Comment:
Qui non ci va `Boolean(checked)`? o va su entrambe o in nessuno dei 2 o mi sbaglio?